smart-campus-webservice-impl/src/main/java/hu/unideb/smartcampus/webservice/api/ejabberd/impl/XmppUserServiceImpl.java

```diff
@@ -37,4 +42,8 @@ public void createUser(String username, String password) throws XmppException {
 
   }
 
+  private String getXmppHost() {
+    return propertyProvider.getPropertyValue(ConfigPropertyKey.SMART_CAMPUS_XMPP_HOST);
```

Comment:
Fine, but you can use @Value annotation to get the mentioned property but it's good too.